modules/graphstudio/pages/map-data-to-graph.adoc

```diff
@@ -87,52 +80,46 @@ If a file is no longer needed, you can remove it from the server by clicking the
 After removing a file from the server, you also need to manually remove data mapping using that file.
 Otherwise, a "file not on server" error will be triggered when loading data.
 
-Continue with <<Map data files to vertex type or edge type>>.
-
-=== Create S3 data source
-
+--
+Amazon S3::
++
+--
 TigerGraph creates a connection to S3 in order to access the file system in an S3 bucket.
 Only one file may be selected at a time.
 
 [NOTE]
-====
 Data files, after decompression, must be in either CSV or Parquet format.
 If a file is in Parquet format, the data you are loading cannot use INT96 data types.
-Date in INT96 types are not loaded into TigerGraph and the respective attribute will be left empty. 
 
 TigerGraph supports loading from archived and compressed S3 files directly.
 Currently supported file extensions include zip, tar.gz, tgz and tar.
 GraphStudio detects the file extension and automatically chooses the corresponding file format.
 If the file is encoded with one of these formats but has a non-standard file extension, you can manually specify the file format.
```

Comment:
S3, GCS, ABS they should all have the same section.

Referring to the GCS section,
> In addition to single files, TigerGraph also supports loading an entire folder by entering the gsutil URI for that folder.
All data files in this folder must share the same data schema.
The folder preview, like the file preview, is limited to the first ten lines of uploaded data.
If a folder contains more than one file and the first file has more than ten lines, only the first ten lines of the first file will appear in the preview.

I think you can union them into one and call it "remote storage" section. The only difference is different ways to make connection. The requirements of the data are the same.